alf/nest/cnest/nest.cpp

```diff
@@ -31,6 +32,15 @@ std::string py_format(const std::string& format, Args... args) {
   return py::cast<py::str>(py_format(format, args...));
 }
 
+py::int_ GetPyInt(unsigned int i) {
+  // cache py::int_ to avoid creating new objects for repeating i
+  static std::map<unsigned int, py::int_> table;
```

Comment:
vector<py::int_> is better for the specific purpose of this file because i won't be big.